### ISSUE TITLE
발표 준비 의논

### ISSUE THREAD

**Issue by bluecri** [17-1-SKKU-OSS/angular-ko#14]
다음 주 화요일이 발표날이여서 준비를 해야 할 것 같아요.
그 때까지 뭘 할지랑, 위키를 어떻게 만들지, 정적 페이지는 어떻게 할지 의논해 봅시다.
저는 그때까지 풀 리퀘스트에 있는거 서로 리뷰해서 퀄리티를 높이려고 노력해야 할 것 같습니다.

**Comment by cshwan78** [comment]
한 풀 리퀘스트당 최소 하나씩은 서로 잘못 된 점을 찾아서 지적하고 그걸 고치는 방향으로 진행하면 어떨까요?

**Comment by cshwan78** [comment]
그리고 정적 페이지를 통해서 발표를 진행하니까 reveal.js 라는 좋은 라이브러리를 사용해서 피피티 형식으로 정적 페이지를 구현해서 발표하시는 분이 편하게 발표할 수 있도록 하는건 어떨까요??

**Comment by bluecri** [comment]
그러면 정적 페이지에는 어떤 내용을 주로 작성하고 위키에는 어떤 내용을 주로 작성할까요?
제 생각에는 위키보다는 정적페이지가 더 자유로우니까 위키에는 프로젝트 진행상황 같은 내용을 커밋처럼 내용을 누적시키고 정적페이지에 발표 관련된 사항이나 팀에 대한 소개 같은 내용을 넣는게 좋을 것 같습니다.

**Comment by cshwan78** [comment]
괜찮은 것 같아요.
저희가 뭘 했는지 직접적인 커밋 내용을 보여주지 않고는 어려울 것 같아보여서 하는 말인데,
커밋 몇 개, 풀리퀘 몇 개, 실제로 풀리퀘 몇 개, 이슈 몇 개, 라인 몇라인, 총 몇 단어 등등 객관적인 수치 데이터를 피피티에 포함 시키는 건 어떨까요? 그리고 그 아래에는 링크를 넣어서 실제로 앵귤러 사이트에서 확인해 볼 수 있도록 하는 것도 좋을 것 같아요.

**Comment by bluecri** [comment]
말씀하신 링크는 저번에 직접 개인 웹 서버에서 실행하셨던 번역 작업을 확인할 수 있은 angularjs 웹페이지인가요?

**Comment by cshwan78** [comment]
네. 그 지금은 잘 사용하지 않아서 계속 서버를 돌려놓고 있지는 않은데, 발표 할 때까지 일주일 정도는 계속 오픈해 둘 수 있을 것 같아요. AWS 서버 학생 크레딧을 받아서 공짜로 사용할 수 있거든요.

**Comment by bluecri** [comment]
위키에는 투두 리스트랑, 번역 관련 저희 만의 커밋 룰들 그리고 CLA상황 같은거를 정리하면 좋을 것 같습니다.

**Comment by cshwan78** [comment]
이전에 설치 할때 설치 하는 방법이 우분투 14.04에서는 잘 적용이 되지 않아 애를 먹었었는데
해당 설치방법 문서를 새로 써서 위키에 저장하고 원래 리포지토리에 이슈를 열어서 사용하도록 권고하는 거는 어떨까요?
이게 설치 방법이 잘못 됬다기 보다는 우분투 14.04 버전에서는 잘 동작하지 않으니까 확실히 동작하는 설치 방법같은거를 추가로 제시하면 좋을 것 같아서요.

**Comment by bluecri** [comment]
그러면 해야할 작업이
공통적으로 서로 풀 리퀘스트에 번역 오류 검사해주기랑.
정적 페이지 관리, reveal.js로 피피티 만들기. 위키 관리, 이렇게 세개 있는거 같은데 세명이서 이를 적당히 나눠서 진행하면 될 것 같습니다.

**Comment by cshwan78** [comment]
위키 관리가 상대적으로 시간이 남을 것 같으니까 원래 리포지토리 관리자에게 메일을 써서 저희 것에 대해 피드백을 달라고 요청하는 건 어떨까요?

**Comment by bluecri** [comment]
그럼 그렇게 3개를 서로 나눠서 진행하는 걸로 하죠. 그 실제로 어떻게 분배할 지는 서로 만나서나 행아웃 같은걸로 진행해 봅시다.
